framework/src/multiapps/MultiApp.C

```diff
@@ -124,9 +124,6 @@ MultiApp::MultiApp(const InputParameters & parameters):
 
   mooseAssert(_input_files.size() == 1 || _positions.size() == _input_files.size(), "Number of positions and input files are not the same!");
 
-  // Fill in the _positions vector
-  fillPositions();
```

Comment:
Definitely not duplicated! This should cause something to break. I'm investigating a little more.